connector/http-agent/app/axios/TenantService.js

```diff
@@ -43,6 +43,10 @@ module.exports = class TenantService {
 
     // Waits for all authentications to complete
     this.tenants = await Promise.all(tenantPromises);
+    this.logger.debug(`All tenants have been loaded`);
+    for (const tenant1 of this.tenants) {
+      this.logger.debug(tenant1.id);
```

Comment:
Ao invés de fazer um loop logando cada tenant em uma linha separada você pode fazer um `this.tenants.join(',')` e passar tudo em um log só. Pode inclusive usar o segundo parâmetro do logger para passar dados extras: `this.logger.debug('message', { tenants: [...] })`